feat: add hh_node measurement (#4137)
influxdata/docs-v2: content/platform/monitoring/influxdata-platform/tools/measurements-internal.md

---

### hh_node (Enterprise only)

> The `hh_node` measurement stores hinted handoff statistics for all queues (shards) for a given node.
> The `hh_node` measurement is available in InfluxDB Enterprise 1.9.8 and later.

Reviewer: ```suggestion
Available in InfluxDB Enterprise 1.9.8 and later.
The `hh_node` measurement stores hinted handoff statistics for all queues (shards) for a given node.

```

---

### hh_node (Enterprise only)
The `hh_node` measurement stores hinted handoff statistics for all queues (shards) for a given node.
The `hh_node` measurement is available in InfluxDB Enterprise 1.9.8 and later.

The `hh_node` measurement has one additional tag:

> - `node` - The destination node for the recorded metrics.

Reviewer: @samhld I'm not clear on action item here, can you expand? thank you!

> - `node` - The destination node for the recorded metrics.

Reviewer: @samhld ah, I see what you mean...I'll add a docs issue

#### queueTotalSize
The sum total number of bytes of disk space used by all hinted handoff queues for a single node.

> This statistics tracks the disk usage of all hinted handoff queues for a given node, not the number
> of bytes waiting to be processed. Due to the implementation of the hinted handoff queue, there will

Reviewer: ```suggestion
Tracks the disk usage of all hinted handoff queues for a given node (not the bytes waiting to be processed). Due to the implementation of the hinted handoff queue, 
```

> be a lag between the bytes being processed and when they are removed from the disk.

Reviewer: ```suggestion
a lag occurs between when bytes are processed and when they're removed from the disk.
```

> `queueTotalSize` is the metric used to determine if a node's hinted handoff queue has reached the
> configured maximum size as given by the [[hinted-handoff]max-size configuration parameter](https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#max-size).

Reviewer: ```suggestion
`queueTotalSize` is used to determine when a node's hinted handoff queue has reached the
maximum size configured in the [hinted-handoff max-size](https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#max-size) parameter.
```
